tests/test_encode_decode.py

```diff
@@ -95,8 +97,92 @@ def test_b85_encode_decode_round_trip(self, payload, pad):
 
 
 class TestBinASCII(unittest.TestCase):
-    # TODO: https://docs.python.org/3/library/binascii.html
-    pass
+    @given(payload=st.binary())
+    def test_b2a_uu_a2b_uu_round_trip(self, payload):
+        x = binascii.b2a_uu(payload)
+        self.assertEqual(payload, binascii.a2b_uu(x))
+
+    @unittest.skipIf(
+        sys.version_info[:2] < (3, 7), "backtick not supported in this library version"
+    )
+    @given(payload=st.binary(), backtick=st.booleans())
+    def test_b2a_uu_a2b_uu_round_trip_with_backtick(self, payload, backtick):
+        x = binascii.b2a_uu(payload, backtick=backtick)
```

Comment:
```suggestion
    @given(payload=st.binary(), backtick=st.booleans())
    def test_b2a_uu_a2b_uu_round_trip_with_backtick(self, payload, backtick):
        if sys.version_info[:2] >= (3, 7):
            x = binascii.b2a_uu(payload, backtick=backtick)
        else:
            x = binascii.b2a_uu(payload)
```

we can still run the test on PyPy's 3.6 implementation, we just ignore the `backtick` argument.

Reply:
Sure 👍